packages/interface/src/components/more-menu-dropdown/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { moreVertical } from '@wordpress/icons';
+
+export default function MoreMenuDropdown( {
```

Comment:
Not convinced about the name of this component. Perhaps it should be something more specific like `OptionsDropdown`. Worth noting there would probably be a future `PreferencesModal`.

Reply:
`OptionsDropdown` sounds good because it matches the dropdown button label.

Reply:
I went for `OptionsMenu` and `OptionsMenuFeatureToggle`. Pretty happy with that 👍